### PR TITLE
Accel 6b test

### PR DESCRIPTION
**Pull request by lianglia-apollo**

### CODE REVIEW

File: modules/canbus/vehicle/lincoln/protocol/accel_6b.cc [hunk 1]
```diff
@@ -26,45 +28,31 @@ const int32_t Accel6b::ID = 0x6B;
...
 }
 
-double Accel6b::vertical_acceleration(const std::uint8_t* bytes,
-                                      int32_t length) const {
-  Byte high_frame(bytes + 5);
+double Accel6b::parse_frames(const std::uint8_t* bytes) const {
+  Byte high_frame(bytes + 1);
   int32_t high = high_frame.get_byte(0, 8);
-  Byte low_frame(bytes + 4);
+  Byte low_frame(bytes + 0);
```
Comment (siyangy) [inline modules/canbus/vehicle/lincoln/protocol/accel_6b.cc]: remove +0 ？

File: modules/canbus/vehicle/lincoln/protocol/accel_6b.cc [hunk 2]
```diff
@@ -26,45 +28,31 @@ const int32_t Accel6b::ID = 0x6B;
...
-  int32_t high = high_frame.get_byte(0, 8);
-  Byte low_frame(bytes + 0);
-  int32_t low = low_frame.get_byte(0, 8);
-  int32_t value = (high << 8) | low;
-  if (value > 0x7FFF) {
-    value -= 0x10000;
-  }
-  return value * 0.010000;
+double Accel6b::lateral_acceleration(const std::uint8_t* bytes) const {
+  return parse_frames(bytes);
```
Comment (startcode) [inline modules/canbus/vehicle/lincoln/protocol/accel_6b.cc]: please check length inside function

File: modules/canbus/vehicle/lincoln/protocol/accel_6b.cc [hunk 3]
```diff
@@ -35,36 +38,29 @@ void Accel6b::Parse(const std::uint8_t* bytes, int32_t length,
...
 
 double Accel6b::vertical_acceleration(const std::uint8_t* bytes,
-                                      int32_t length) const {
-  Byte high_frame(bytes + 5);
+                                      const int length) const {
+  DCHECK_EQ(length, CANBUS_MESSAGE_LENGTH);
+  return parse_double_frames(bytes + 4, 2);
+}
+
+double Accel6b::parse_double_frames(const std::uint8_t* bytes,
```
Comment (startcode) [inline modules/canbus/vehicle/lincoln/protocol/accel_6b.cc]: rename to parse_two_framesi f it is dedicated to use two frames

Comment (startcode) [inline modules/canbus/vehicle/lincoln/protocol/accel_6b.cc]: use two arguments, such as high bytes and low_bytes to avoid check length

**Pull request merged**